examples/tutorials/quantization_quick_start_mnist.py

```diff
@@ -82,8 +87,8 @@
 # %%
 # build tensorRT engine to make a real speedup, for more information about speedup, please refer :doc:`quantization_speedup`.
 
-from nni.compression.pytorch.quantization_speedup import ModelSpeedupTensorRT
-input_shape = (32, 1, 28, 28)
-engine = ModelSpeedupTensorRT(model, input_shape, config=calibration_config, batchsize=32)
-engine.compress()
-test_trt(engine)
+# from nni.compression.pytorch.quantization_speedup import ModelSpeedupTensorRT
```

Comment:
why

Reply:
the machine that supports tensorrt has expired, will try to add back before release

Reply:
lucky... I found we have tensorrt pipeline, so revert it...